src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -86,30 +84,6 @@ public class TreeMojo extends AbstractMojo {
     @Parameter(property = "outputEncoding", defaultValue = "${project.reporting.outputEncoding}")
     private String outputEncoding;
 
-    /**
-     * Contains the full list of projects in the reactor.
-     */
-    @Parameter(defaultValue = "${reactorProjects}", readonly = true, required = true)
```

Comment:
does this cause problems for existing projects that set these?

Reply:
no problem ... like unused variables are